scripts/testnet/lynx_config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "LynxRootApplication": {},
+  "LynxTACoChildApplication": {
+    "_rootApplication": "$LynxRootApplication"
+  },
+  "LynxRitualToken": {
```

Comment:
Can we not smurf name here? In addition, I think it would be best if there's a common schema for these files, and therefore the keys being used are "standard" (e.g. `rootApplication`, `childApplication`, etc)
Also, what's the RitualToken? Is it the ritual initiation payment token?

Reply:
> Also, what's the RitualToken? Is it the ritual initiation payment token?
Yes. It's a made up ERC-20 to use on Lynx for currency i.e. paying for rituals.